chemprop/features/features_generators.py

```diff
@@ -131,7 +131,9 @@ def rdkit_2d_normalized_features_generator(mol: Molecule) -> np.ndarray:
         """Mock implementation raising an ImportError if descriptastorus cannot be imported."""
         raise ImportError('Failed to import descriptastorus. Please install descriptastorus '
                           '(https://github.com/bp-kelley/descriptastorus) to use RDKit 2D normalized features.')
-
+except AttributeError as e:
+    raise AttributeError('`descriptastorus==2.6.1` is incompatible with `scipy<1.9`. Please try changing '
+                         'descriptastorus versions or update scipy to avoid issues with `scipy.stats.gibrat`.')
```

Comment:
if you want to catch and re-throw the same exception:
```suggestion
except AttributeError as e:
    raise e('`descriptastorus==2.6.1` is incompatible with `scipy<1.9`. Please try changing '
                         'descriptastorus versions or update scipy to avoid issues with `scipy.stats.gibrat`.')
```
This is useful for the case where the exception to catch changes in different version, so a dev changes what to catch but then forgets to change the exception that's thrown afterwards. It's a minor nit, so you can take it or leave it.

Reply:
as a side note: is it only version `2.6.1` that this error appears in, or is it anything $\geq$`2.6.1`?

Reply:
2.6.1 is the latest version. I hope that in the next version descriptastorus will fix the problem. I have opened an issue on their repo. I see the error is also in 2.5.1 (but not 2.5.0) so I'll update this PR.

Reply:
The commit to change `AttributeError("` to `e("` gave `TypeError: 'AttributeError' object is not callable` so I changed it again.